specification/ai/Azure.AI.Unified/models/models.tsp

```diff
@@ -0,0 +1,73 @@
+import "@typespec/rest";
+import "@azure-tools/typespec-autorest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "../main.tsp";
+import "../common/models.tsp";
+import "@typespec/openapi";
+
+using TypeSpec.OpenAPI;
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.Core.Traits;
+
+namespace Azure.AI.Unified;
+
+#suppress "@azure-tools/typespec-azure-core/composition-over-inheritance"
+@doc("Model Deployment Definition")
+model Model {
+  @doc("Name of the deployment for the model")
+  @visibility("read")
+  name: string;
+
+  @doc("Publisher-specific name of the model")
+  @visibility("read")
+  modelName: string;
+
+  @doc("Publisher-specific version of the model")
+  @visibility("read")
+  modelVersion: string;
+
+  @doc("Name of the model publisher")
+  @visibility("read")
+  publisher: string;
+
+  @doc("Capabilities of model")
+  @visibility("read")
+  capabilities: Record<string>;
+
+  @doc("RAI policy enabled for model")
+  @visibility("read")
+  raiPolicyName: string;
+
+  @doc("Sku of the model deployment")
+  @visibility("read")
+  sku: Sku;
+
+  @doc("System data of the resource")
+  @visibility("read")
+  systemData: SystemData;
```

Comment:
Why is this needed? How will this be used? I prefer we don't add it, until we see customer ask for it.